TradeItIosTicketSDK2/TradeItAlertManager.swift

```diff
@@ -1,9 +1,15 @@
 import UIKit
 
 @objc public class TradeItAlertManager: NSObject {
+    public static let sharedInstance = TradeItAlertManager()
+
+    var alertClosureQueue: [() -> Void] = []
```

Comment:
We store closures that will generate the alert with the correct viewController etc that it was originally called with.